examples/porcelain/main.go

```diff
@@ -48,10 +49,7 @@ func highLevelExample() error {
 	ctx := context.Background()
 
 	// Logging is important - replace with your own zap logger
-	logger, err := zap.NewDevelopment()
-	if err != nil {
-		return err
-	}
+	logger := slog.New(slog.NewTextHandler(os.Stderr, &slog.HandlerOptions{Level: slog.LevelDebug}))
```

Comment:
This comment mentions zap

Reply:
Latest commit removes zap from the comments